bundle/edu.gemini.itc/src/main/resources/sed/non_stellar/Mars.nm

```diff
@@ -0,0 +1,457 @@
+# Mars
```

Comment:
Ahhh... I think I see. Do these `nm` files work with the ITC servlet, making my previous questions moot?

Reply:
General note: with the addition of the solar-system SEDs, do we need to regenerate `baseline.txt`?

Reply:
The ITC tests ran without errors, so I don't think the baseline.txt needs to be updated.